lib/ts/recipeImplementation.ts

```diff
@@ -199,7 +199,8 @@ export default function RecipeImplementation(recipeImplInput: {
                 if (!validationRes.isValid) {
                     errors.push({
                         validatorId: validator.id,
-                        reason: validationRes.reason
+                        reason: validationRes.reason,
+                        redirectPath: validationRes.redirectPath
                     });
                 }
```

Comment:
We should add a test to enforce that the claims sent to `validateClaims` array are indeed checked in the same order as they are in the array. Which means that the error[] returned should have the errors in the same order as well.

Reply:
done in #143